extensions/charts/vsphereoutoftree.go

```diff
@@ -124,17 +144,17 @@ func InstallVsphereOutOfTreeCharts(client *rancher.Client, vsphereTemplate *node
 }
 
 // vsphereCPIChartInstallAction is a helper function that returns a chartInstallAction for aws out-of-tree chart.
```

Comment:
This comment looks like it was copy paste since it says `aws out-of-tree`. This needs to be updated for vsphere.